django/cantusdb_project/main_app/templates/403.html

```diff
@@ -8,6 +8,9 @@ <h5>Access denied</h5>
             <br>
             <dl>
                 <dd>You are not authorized to access this page.</dd>
+                {% if not user.is_authenticated %}
+                    <dd>This could be because you are not currently logged in. If you want, you can try <a href="{% url 'login' %}?next={{ request.path }}">logging in</a>.</dd>
```

Comment:
One minor suggestion: Perhaps we can change the wording here. Make it clear that only contributors should log in, so that general visitors who are not contributors (i.e., who don't have an account) don't ask us about registration. Maybe change "if you want" to "if you are a contributor to the database".

Reply:
Thank you, good suggestion. I've added another commit addressing this.